src/components/dls/QuranWord/QuranWord.tsx

```diff
@@ -64,18 +67,14 @@ const QuranWord = ({
 
   const [isTooltipOpened, setIsTooltipOpened] = useState(false);
   const { showWordByWordTranslation, showWordByWordTransliteration } = useSelector(
-    selectWordByWordByWordPreferences,
+    selectWordByWordPreferences,
     shallowEqual,
   );
   const readingPreference = useSelector(selectReadingPreference);
   const showTooltipFor = useSelector(selectShowTooltipFor, areArraysEqual);
 
-  // creating wordLocation instead of using `word.location` because
-  // the value of `word.location` is `1:3:5-7`, but we want `1:3:5`
-  const wordLocation = makeWordLocation(word.verseKey, word.position);
```

Comment:
@muhajirdev is removing this safe?

Reply:
I think @naveed-ahmad updated the API. so `word.location` is already using the expected format now, so we can remove `wordLoation`